src/themes/default/components/core/AddToCart.vue

```diff
@@ -19,7 +19,7 @@ export default {
       this.$forceUpdate()
     },
     canBeAdded (product) {
-      return formatProductMessages(product.errors) !== ''
+      return formatProductMessages(product.errors) !== '' || this.disabled
```

Comment:
I believe this should be an opposite - can be added if this is *not* disabled
```suggestion
      return !disabled && !formatProductMessages(product.errors)
```